--- asyncio-3.4.3/tests/test_events.py~	2015-02-12 16:27:23.000000000 +0100
+++ asyncio-3.4.3/tests/test_events.py	2016-05-07 13:01:03.741905201 +0200
@@ -1227,6 +1227,7 @@
         # extra info is available
         self.assertIsNotNone(proto.transport.get_extra_info('pipe'))
 
+    @unittest.skipUnless(False, 'Builders do not provide ptys')
     @unittest.skipUnless(sys.platform != 'win32',
                          "Don't support pipes for Windows")
     # select, poll and kqueue don't support character devices (PTY) on Mac OS X
@@ -1329,6 +1330,7 @@
         self.loop.run_until_complete(proto.done)
         self.assertEqual('CLOSED', proto.state)
 
+    @unittest.skipUnless(False, 'Builders do not provide ptys')
     @unittest.skipUnless(sys.platform != 'win32',
                          "Don't support pipes for Windows")
     # select, poll and kqueue don't support character devices (PTY) on Mac OS X
